scripts/update-rules.js

```diff
@@ -1,18 +1,24 @@
-'use strict';
+import fs from 'node:fs';
+import { createRequire } from 'node:module';
+import path, { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import prettier from 'prettier';
 
-const fs = require('fs');
-const path = require('path');
+import a11y from '../lib/config/a11y.js';
+import recommended from '../lib/config/recommended.js';
+import stylistic from '../lib/config/stylistic.js';
+import rules from '../lib/rules/index.js';
```

Comment:
Had to convert this script to ESM because it imports the rules index which is ESM.